CarDealer/Interfaces/ICarBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarDealer.Models
+{
+    public interface ICarBuilder
+    {
+        void SetModel(string Model);
+        void SetColor(string Color);
+        void SetEngine(string Engine); // asta cred ca trebuie EFuelType, nu string
```

Comment:
Daca te uiti pe diagrama, acolo tipul era Engine, nu string si nici EFuelType. Pune numele la parametri cu litera mica.

Reply:
O sa ai si ceva conflicte in fisierul .csproj si nu o sa te lase sa dai merge, spune-mi cand esto gata si ne uitam cum le rezolvam